addons/mail/static/tests/qunit_suite_tests/systray_activity_menu_tests.js

```diff
@@ -113,63 +80,66 @@ QUnit.test('activity menu widget: activity menu with 3 records', async function
     };
     await testUtils.dom.click(activityMenu.$('.dropdown-toggle'));
     assert.hasClass(activityMenu.$el, 'show', 'ActivityMenu should be open');
-    await testUtils.dom.click(activityMenu.$(".o_activity_filter_button[data-model_name='Issue'][data-filter='overdue']"));
+    await testUtils.dom.click(activityMenu.$(".o_activity_filter_button[data-model_name='res.country'][data-filter='overdue']"));
     assert.doesNotHaveClass(activityMenu.$el, 'show', 'ActivityMenu should be closed');
     // case 2: click on "today"
     context = {
         force_search_count: 1,
         search_default_activities_today: 1,
     };
     await testUtils.dom.click(activityMenu.$('.dropdown-toggle'));
-    await testUtils.dom.click(activityMenu.$(".o_activity_filter_button[data-model_name='Issue'][data-filter='today']"));
+    await testUtils.dom.click(activityMenu.$(".o_activity_filter_button[data-model_name='res.country'][data-filter='today']"));
     // case 3: click on "future"
     context = {
         force_search_count: 1,
         search_default_activities_upcoming_all: 1,
     };
     await testUtils.dom.click(activityMenu.$('.dropdown-toggle'));
-    await testUtils.dom.click(activityMenu.$(".o_activity_filter_button[data-model_name='Issue'][data-filter='upcoming_all']"));
+    await testUtils.dom.click(activityMenu.$(".o_activity_filter_button[data-model_name='res.country'][data-filter='upcoming_all']"));
     // case 4: click anywere else
     context = {
         force_search_count: 1,
         search_default_activities_overdue: 1,
         search_default_activities_today: 1,
     };
     await testUtils.dom.click(activityMenu.$('.dropdown-toggle'));
-    await testUtils.dom.click(activityMenu.$(".o_mail_systray_dropdown_items > div[data-model_name='Issue']"));
+    await testUtils.dom.click(activityMenu.$(".o_mail_systray_dropdown_items > div[data-model_name='res.country']"));
```

Comment:
country doesn't have the `mail.activity.mixin`, should use res.partner or move test to test_mail with a fake server model